client/src/assets/css/styles.css

```diff
@@ -527,26 +529,50 @@ table {
    
  }
  
+ .wavecontainer {
+  /* Wrapper around Waves and Slogan */
+  
+  /* display: flex;
+  flex-direction: column;
+  flex: 100%; */
+ }
+
  .slogan {
-   text-align:center;
```

Comment:
Flipped Waves from on bottom of Slogan to above and on bottom of navigation to simplify page content